source/MaterialXCore/Interface.cpp

```diff
@@ -332,6 +332,10 @@ bool Input::validate(string* message) const
 
     if (hasDefaultGeomPropString())
     {
+        if (!parent->isA<NodeDef>())
```

Comment:
@jstone-lucasfilm , @dbsmythe. This should fix the defaultgeomprop test to be compliant.